app/views/design-system/rsd-design-system/design-patterns/footer/index.html

```diff
@@ -0,0 +1,176 @@
+          href="https://educationgovuk.sharepoint.com/sites/lvewp00299/SitePages/Prepare-Conversions-and-Transfers.aspx">Prepare
+          conversions and transfers - SharePoint</a></li>
+    </ul>
+
+    <h2>Links</h2>
+    <p>The RSD footer includes several links:</p>
+    <h3>Email Service Support for help with using this system</h3>
+    <p>This is a mail-to link with the subject header:</p>
+    <code>[Product name]: support query</code>
+    <p>For example, Prepare conversions and transfers: support query</p>
+    <p> It has this text underneath the link: "We aim to respond within 5 working days, or one working day for
+      more urgent queries." </p>
+    <h3>How to use this system (opens in a new tab)</h3>
+    <p> This links to the section of a product's SharePoint page with guidance on how to use the product. For
+      example, <a
+        href="https://educationgovuk.sharepoint.com/sites/lvewp00299/SitePages/Prepare-Conversions-and-Transfers.aspx#guidance-for-using-prepare-conversions-and-transfers">Guidance
+        for using Prepare conversions and
+        transfers</a>. </p>
+    <p>If you don't have SharePoint guidance about how to use your product, you do not need to include this
+      hyperlink.</p>
+    <h3>Give feedback (opens in a new tab)</h3>
+    <p>This links to a Microsoft Form where users can give feedback about the product. The same Microsoft Form
+      is used across all RSD products: <a
+        href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-SZtRygfwTNOqcfRq-MXpv9UOTIyQlNYR0hJT1Q0TUFVSlJGVFhES01LVC4u">Give
+        feedback about a Regions Group system - Microsoft Form</a>.
+    </p>
+    <h3>Suggest a change (opens in a new tab)</h3>
+    <p>This links to a Microsoft Form, which is the same across all products: <a
+        href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-fkHK2JGo_BIpVChpLMaBFpUNUFDSzhQN0FHVklTV0JWTzFZTjNKWTNJUi4u">Suggest
+        a change to a Regions Group system - Microsoft Form</a>.
+    </p>
+    <h3>Accessibility statement</h3>
+    <p>This links to the product's accessibility statement.</p>
+    <h3>Cookies</h3>
+    <p>This links to the product's cookies page.</p>
+    <h3>Privacy notice</h3>
+    <p>This links to the product's privacy notice.</p> 
+    <p>If the product does not need a privacy notice, you do not need to include this link.</p>
+    <h3>Crown copyright</h3>
+    <p>This links to <a
+      href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">Crown copyright - Re-using PSI</a>.
+    <h2>Research on this component</h2>
+    <p>
+      We have not done research on this component, but it is being used to meet these needs from our <a
+        href="https://educationgovuk.sharepoint.com/sites/ServiceDeliveryDirectorate/Lists/RSD%20crossprogramme%20user%20needs/AllItems.aspx?env=WebViewList">user
```

Comment:
Internal link again

Reply:
We added the text "(opens in DfE Microsoft List)" now.